static-analysis/teammates-eslint-es6.yml

```diff
@@ -20,6 +20,15 @@ rules:
     # enforce a maximum line length
     max-len: ["error", 125]
 
+    # disallow continue statements
+    no-continue: "off"
+
+    # disallow mixes of different operators
+    no-mixed-operators: "off"
```

Comment:
`no-param-reassign`: I agree that function parameters are not to be re-assign-able (in fact it will still be flagged), but the Airbnb rule even flags things such as:
```javascript
function (a) {
    a.foo = 'foo';
}
```